tests/system/feature_store/test_spark_engine.py

```diff
@@ -921,9 +921,17 @@ def test_get_offline_features_with_filter_and_indexes(self):
         assert resp_df.equals(target_df)
         assert resp_df[["bad", "department"]].equals(expected_df)
 
-    # ML-2802
-    @pytest.mark.parametrize("passthrough", [True, False])
-    def test_get_offline_features_with_spark_engine(self, passthrough):
+    # ML-2802, ML-3397
+    @pytest.mark.parametrize(
+        ["target_type", "passthrough"],
+        [
+            (ParquetTarget, False),
+            (ParquetTarget, True),
+            (CSVTarget, False),
+            (CSVTarget, True),
+        ],
+    )
+    def test_get_offline_features_with_spark_engine(self, passthrough, target_type):
```

Comment:
Where do you use `target_type`, seems to me that you are just using `ParquetTarget` explicitly

Reply:
Thanks, well, that's disappointing. I'll fix and rerun the test.